src/components/ScholarshipAmountField.jsx

```diff
@@ -21,6 +25,10 @@ const useStyles = makeStyles({
 function ScholarshipAmountField(props) {
   const classes = useStyles();
   const {
+    typeError,
+    minAmountError,
+    maxAmountError,
```

Comment:
Simplify by simply passing `error` and calculate individual fields' `error` (and `helperText` too if you like) based on `amountType`
```suggestion
    error
```
Right beneath this statement you can do:
```jsx
const isRange = amountType === AmountType.Range;
const isFixed = amountType === AmountType.Fixed;
```

And then in your `switch` statement below render text fields like this:
```jsx
<AmountTextField error={error && isRange}
              value={minAmount || ''}
              onChange={...}
              disabled={!isRange}
              helperText={helperText && isRange}/>
```